### PR TITLE
add InitConfigure of ServiceBus

### PR DESCRIPTION
**Pull request by muxuelan**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
Add InitConfigure of ServiceBus for further development.
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: edge/pkg/servicebus/config/config.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package config
+
+import (
+"sync"
+
+"github.com/kubeedge/kubeedge/pkg/apis/componentconfig/edgecore/v1alpha1"
```
Comment (fisherxu) [inline edge/pkg/servicebus/config/config.go:6]: Run gofmt here :)

Comment (muxuelan) [reply edge/pkg/servicebus/config/config.go]: Okay, got it~

File: edge/pkg/servicebus/config/config.go [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+)
+
+var Config Configure
+var once sync.Once
+
+type Configure struct {
+	v1alpha1.ServiceBus
+}
+
+func InitConfigure(sb *v1alpha1.ServiceBus) {
```
Comment (fisherxu) [inline edge/pkg/servicebus/config/config.go:16]:
```suggestion
func InitConfigure(s *v1alpha1.ServiceBus) {
```
`s` is fine here....

**Comment by muxuelan** [comment]
Oh~

**Pull request closed without merging**